File: php/RawDataStore.php
<?php

/**
 * Description of RawDataStore
 *
 * @author paul
 */
class RawDataStore extends DataStore {
  function __construct() {
    parent::__construct('raw');
  }

  public function set_data_dir($data_dir) {
    parent::set_data_dir($data_dir . '/raw');
  }
}

?>

File: java/lib/src/main/java/com/caffeinatedbliss/kayveep/RawDataStore.java
package com.caffeinatedbliss.kayveep;

import java.io.File;

/**
 * @author Paul S. Hawke (paul.hawke@gmail.com)
 *         On: 2/22/12 at 11:36 PM
 */
public class RawDataStore extends DataStore {
	public RawDataStore() {
		super("raw");
	}

	@Override
	public void setDataDir(String dir) {
		super.setDataDir(new File(dir, "raw").getAbsolutePath());
	}
}
